wavespectra/input/__init__.py

```diff
@@ -64,12 +74,18 @@ def open_netcdf_or_zarr(filename_or_fileglob, file_format, mapping={}, chunks={}
         - dset (Dataset): spectra dataset object read from ww3 file.
 
     """
-    # Allow chunking using wavespctra names
+    # Allow chunking using wavespectra names
     _chunks = chunks_dict(chunks, mapping)
     if file_format == "netcdf":
-        dset = xr.open_mfdataset(
-            filename_or_fileglob, chunks=_chunks, combine="by_coords"
-        )
+        # Use mfdataset to open if a filename or fileglob is passed
```

Comment:
Yes something like that. I thought of using `xr.open_mfdataset` if `filename_or_fileglob` is an iterable, `xr.open_dataset` otherwise. Another other option is to simply put a try/except looking for `IndexError` which is what seems to be raised when using open_mfdataset on a file-object, and use open_dataset in that case.

Reply:
Cool, I like the try except approach better too